sample/OSCON17/speakers

```diff
@@ -0,0 +1,622 @@
+      "featured":false,
+      "sessions":[
+         {
+            "title":"Step 1:Punch a tree",
+            "id":2
+         }
+      ],
+      "city":"",
+      "gender":"male",
+      "heard_from": "",
+      "icon":"",
+      "small":"",
+      "speaking_experience":"",
+      "sponsorship_required":"",
+      "thumbnail":"https://raw.githubusercontent.com/mahikaw/Fossasia-sample-images/master/evan%20booth.jpg"
+   },
+   {
+      "id":4,
+      "name":"Christopher Aedo",
+      "email":"",
+      "mobile":"",
+      "photo":"https://raw.githubusercontent.com/mahikaw/Fossasia-sample-images/master/christopher%20aedo.jpg",
+      "organisation":"IBM",
+      "position":"",
+      "country":"",
+      "short_biography":"",
+      "long_biography":"Christopher Aedo is a developer advocacy program manager at IBM, where he leads an amazing team of upstream developers. Christopher has been working with and contributing to open source software since his college days. When he’s not at work or speaking at a conference, he’s probably using a Raspberry Pi to brew and ferment a tasty home brew in Portland, OR.",
+      "website":"http://ibm.com/",
+      "twitter":"https://twitter.com/docaedo",
+      "facebook":"",
+      "github":"",
+      "linkedin":"",
+      "featured":false,
+      "sessions":[
+         {
+            "title":"Why choose open infrastructure?",
+            "id":3
+         }
+      ],
+      "city":"",
+      "gender":"male",
+      "heard_from": "",
+      "icon":"",
+      "small":"",
+      "speaking_experience":"",
+      "sponsorship_required":"",
+      "thumbnail":"https://raw.githubusercontent.com/mahikaw/Fossasia-sample-images/master/christopher%20aedo.jpg"
+   },
+   {
+      "id":5,
+      "name":"Dianne Marsh",
+      "email":"",
+      "mobile":"",
+      "photo":"https://drive.google.com/uc?export=download&id=0B2b0W4GqpgLFTE1BbXppSkt1Vlk",
+      "organisation":"Netflix",
+      "position":"",
+      "country":"",
+      "short_biography":"",
+      "long_biography":"Dianne Marsh is a director of engineering at Netflix, where she leads a team responsible for tools and systems used for continuous integration, delivery, and deployment to the AWS cloud by nearly all engineers in the company—which are often released as open source tools to the broad community. Dianne coauthored Atomic Scala with Bruce Eckel. She holds a master of science degree in computer science from Michigan Technological University.",
+      "website":"http://netflix.com/",
+      "twitter":"https://twitter.com/dmarsh",
+      "facebook":"",
+      "github":"",
+      "linkedin":"",
+      "featured":false,
+      "sessions":[
+         {
+            "title":"How and why Netflix drives industry engagement",
+            "id":4
+         }
+      ],
+      "city":"",
+      "gender":"male",
+      "heard_from": "",
+      "icon":"",
+      "small":"",
+      "speaking_experience":"",
+      "sponsorship_required":"",
+      "thumbnail":""
```

Comment:
provide the thumbnails as well. use the local images instead of providing the online links.